services/src/main/java/org/keycloak/protocol/oid4vc/issuance/signing/VCSigningServiceProviderFactory.java

```diff
@@ -38,6 +38,12 @@
  * @author <a href="https://github.com/wistefan">Stefan Wiedemann</a>
  */
 public interface VCSigningServiceProviderFactory extends ComponentFactory<VerifiableCredentialsSigningService, VerifiableCredentialsSigningService>, EnvironmentDependentProviderFactory {
+
+    /**
+     * Key for the realm attribute providing the issuerDidy.
+     */
+    String ISSUER_DID_REALM_ATTRIBUTE_KEY = "issuerDid";
```

Comment:
When using an realm attribute, how about declaring it in `RealmAttributes`?
@mposolda WDYT?

Reply:
I took a look at that, but it would require to put a dependency to the ```keycloak-model-jpa``` to the ```keycloak-services``` module.

Reply:
@wistefan I see. Thank you for the clarification.

Reply:
@mposolda Yes I will agree with you.